packages/financial-templates-lib/src/logger/SpyTransport.js

```diff
@@ -22,29 +21,27 @@ class SpyTransport extends Transport {
 // Helper function used by unit tests to check if the last message sent to winston contains a particular string value.
 // Caller feeds in the spy instance and the value to check.
 function lastSpyLogIncludes(spy, value) {
-  // Sinon's getCall(n) function returns values sent in the nth (zero-indexed) call to the spy. Check both the mrkdown and message sent.
-  const lastReturnedArgMrkdwn = spy.getCall(-1).lastArg.mrkdwn.toString();
-  const lastReturnedArgMessage = spy.getCall(-1).lastArg.message.toString();
-  return lastReturnedArgMrkdwn.indexOf(value) !== -1 || lastReturnedArgMessage.indexOf(value) !== -1;
+  // Sinon's getCall(n) function returns values sent in the nth (zero-indexed) call to the spy. Flatten the whole object
```

Comment:
change made to enable the `lastSpyLogIncludes` to check for strings within an error message. previously this only looked in the markdown and message. now it includes everything in the Winston log, including errors.